### PR TITLE
Add a plugin readme file

### PR DESCRIPTION
**Pull request by kjellr**
Follows the formatting and guidelines for [WordPress Plugin Directory readmes](https://wordpress.org/plugins/developers/#readme). 

Copy edits and suggestions are welcome!

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+License: GPLv2 or later
+License URI: https://www.gnu.org/licenses/gpl-2.0.html
+
+A plugin for trying out design experiments in WP-Admin. Brought to you by the WordPress Design Team.
+
+== Description ==
+
+The Design Experiments plugin is a tool for prototyping designs within the context of WP-Admin. It contains a number of proof-of-concepts that the [WordPress Design Team](http://make.wordpress.org/design/) is exploring. 
+
+An important disclaimer: This plugin is called experimental for a reason. It is for testing concepts, and is not intended for use on a production site. These experiments may hide or visually break functionality in the admin area. Some of the experiments may break after a while, and will be removed from the plugin.
```
Comment (mapk) [inline readme.txt]: Can we add something like, "Do not rely on this plugin for permanent changes to your site." to the end of this paragraph.

File: readme.txt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+== Frequently Asked Questions ==
+
+= What do I do if an experiment appears broken? =
+
+These are rough proof-of-concepts, so some breakage is generally expected. If you'd like to follow up with the team who built the plugin though, you may [submit an issue on GitHub](https://github.com/WordPress/design-experiments/issues). 
+
+= I have an idea for an experiment, how do I submit it? =
+
+If you'd like to build your own experiment, or just submit an idea, you can do so by at [the plugin's GitHub page](https://github.com/WordPress/design-experiments).
```
Comment (mapk) [inline readme.txt]: Let's drop the word "at" in this sentence.

File: readme.txt (hunk 1 above)
Comment (kjellr) [reply readme.txt]: 👍 Added

File: readme.txt (hunk 2 above)
Comment (kjellr) [reply readme.txt]: I revised this a little bit to: 

> There are instructions for submitting ideas and building your own experiments at [the plugin's GitHub page](https://github.com/WordPress/design-experiments).

**Pull request merged**